lib/fflip.js

```diff
@@ -239,8 +223,20 @@ var self = module.exports = {
 			return;
 		}
 
-		getFeatures(function(data) {
-			getFeaturesCallback(data, callback);
+		getFeatures(function(err, data) {
+			if(typeof err === 'undefined') {
```

Comment:
Some libraries pass `null` when there's no error (eg. `require('util').callbackify`). Any falsy error should be ignored, really:

```suggestion
			if(!err) {
```